### DISCUSSION TITLE
增加一个选项, 允许用户选择, 当点击翻译按钮时, 是优先全文翻译, 还是优先清除翻译.

### DISCUSSION THREAD

**Discussion by XMB-7** [immersive-translate/immersive-translate#1371] [Ideas]
### 新功能 | New Feature

如题

### 为什么您想要添加这些功能 | Why you want to add these features

对我而言, 绝大多数时间都在使用悬浮翻译或者说是局部翻译, 哪里没看懂, 哪里翻译一下也就完了. 
但是当我想要一键清除网页中的少数译文时, 却发现首先触发的是全文翻译, 然后二次点击才能实现译文清除, 这是不太友好的. 
换而言之, 现在的问题是, 消除译文这一功能只跟全局翻译绑定, 即便已触发悬浮翻译, 当点击消除译文功能按钮时(或触发相应快捷键时), 依然因为没有进行过全局翻译而首先进行全局翻译, 待全局翻译后再二次点击才能真正实现消除译文, 而不会因为已经进行过悬浮翻译而直接进行消除译文. 

### 提议 | Proposal

添加一个选项设置, 当页面内有局部译文时, 允许用户点击翻译功能按钮优先触发消除译文.
或者按修饰键(比如ctrl)点击翻译功能按钮强制进行译文清除, 又或者单独为清除译文功能单独设一个快捷键.

**Comment by theowenyoung** [comment]
在使用鼠标悬停的时候，再次按下你设定的修饰键【Ctrl】，即可还原那段的内容。

这个可以满足你的需求吗？

**Reply by XMB-7** [reply]
其实现在就是你说的这样, ctrl+hover触发悬浮翻译后, 再次ctrl+hover即可还原那段内容.
真正的问题在于, 比如一个文章有20个段落, 假设我悬浮翻译了其中的第1, 10, 20段, 那么当我想要还原这三段的内容时, 我需要从文章头滑到文章尾在指定段落上按三次ctrl, 这是非常麻烦的, 而我期待的是, 按一下翻译按钮, 直接把整篇文章已有的悬浮翻译译文都给隐藏掉. (这还是拿悬浮翻译了三段举例, 可想假如我翻译了其中的15个段落呢.) 但现在的行为是, 按一下翻译按钮, 它把剩余的17个段落也翻译了一遍, 二次点击翻译按钮, 才会消除全文的译文, 不仅浪费token, 而且操作不够便捷.

**Reply by theowenyoung** [reply]
那其实直接按【cmd+r]刷新下页面也可以..

**Reply by XMB-7** [reply]
刷新页面的确是一种选择, 但少数时候刷新会丢失对当前页面的阅读进度.
